templates/components/products/options/swatch.html

```diff
@@ -59,19 +59,23 @@
                     <span title="{{this.label}}" style="background-color: #{{data.[0]}}"></span>
                     <span title="{{this.label}}" style="background-color: #{{data.[1]}}"></span>
                     <span title="{{this.label}}" style="background-color: #{{data.[2]}}"></span>
+                    <span>{{this.label}}</span>
```

Comment:
Couldn't you use `for` loop and display as many labels as elements in `data` ? 🤔 
I know that the `if` wasn't created now, but seems like it could be simplified I think

Reply:
up, seems like you've missed this one :)